### PR TITLE
Added ReXNet v1 1.0x model

### PR DESCRIPTION
**Pull request by Anna-Mironova**
TODO:

- [x]  Add GFLOPs and MParams

### CODE REVIEW

File: models/public/rexnet-v1-1.0x/rexnet-v1-1.0x.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+# rexnet-v1-1.0x
+
+## Use Case and High-Level Description
+
+ReXNet V1 1.0x is image classification model pretrained on ImageNet dataset. Rank eXpansion Network(ReXNet): Diminishing Representational Bottleneck on Convolutional Neural Network.
```
Comment (eaidova) [inline models/public/rexnet-v1-1.0x/rexnet-v1-1.0x.md:5]: this text is not logically connected, of I right understand, `Rank eXpansion Network(ReXNet): Diminishing Representational Bottleneck on Convolutional Neural Network.` it is name of paper.

File: models/public/rexnet-v1-1.0x/accuracy-check.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+models:
+  - name: rexnet-v1-1.0x
+
```
Comment (eaidova) [inline models/public/rexnet-v1-1.0x/accuracy-check.yml:3]: please provide config for onnx_runtime if it is possible to evaluate

File: demos/classification_demo/models.lst [hunk 3]
```diff
@@ -50,6 +50,7 @@ resnet-50-pytorch
 resnet-50-tf
 resnet18-xnor-binary-onnx-0001
 resnet50-binary-0001
+rexnet-v1-1.0x
```
Comment (eaidova) [inline demos/classification_demo/models.lst:53]: I think it better to align names for width multiplier between different models (x1.0 in shufflenet used)

Comment (Anna-Mironova) [reply demos/classification_demo/models.lst]: Done

File: models/public/rexnet-v1-1.0x/rexnet-v1-1.0x.md (hunk 1 above)
Comment (Anna-Mironova) [reply models/public/rexnet-v1-1.0x/rexnet-v1-1.0x.md]: Updated, please take a look

File: models/public/rexnet-v1-x1.0/rexnet-v1-x1.0.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+## Use Case and High-Level Description
+
+ReXNet V1 x1.0 is image classification model pretrained on ImageNet dataset. Rank eXpansion Network (ReXNet) is a network derived from research to mitigate the representational bottleneck in a network.
+
+The model input is a blob that consists of a single image of "1x3x224x224" in RGB order.
+
+The model output is typical object classifier for the 1000 different classifications  matching with those in the ImageNet database.
+
+For details see [repository](https://github.com/clovaai/rexnet) and [paper](https://arxiv.org/pdf/2007.00992.pdf).
```
Comment (eaidova) [inline models/public/rexnet-v1-x1.0/rexnet-v1-x1.0.md:11]:
```suggestion
ReXNet V1 x1.0 is network from Rank eXpansion Network (ReXNet) models family, derived from research to mitigate the representational bottleneck. it is image classification model pretrained on ImageNet dataset.

The model input is a blob that consists of a single image of "1x3x224x224" in RGB order.

The model output is typical object classifier for the 1000 different classifications  matching with those in the ImageNet database.

For details see [repository](https://github.com/clovaai/rexnet) and [paper](https://arxiv.org/pdf/2007.00992.pdf).
```

File: models/public/rexnet-v1-x1.0/model.yml [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+conversion_to_onnx_args:
+  - --model-path=$dl_dir
+  - --model-name=ReXNetV1
+  - --weights=$dl_dir/rexnetv1_1.0x.pth
+  - --import-module=rexnetv1
+  - --model-param=width_mult=1.0
+  - --input-shape=1,3,224,224
+  - --output-file=$conv_dir/rexnet-v1-x1.0.onnx
+  - --input-names=input.1
+  - --output-names=1424
```
Comment (eaidova) [inline models/public/rexnet-v1-x1.0/model.yml:54]: I think named output is better that number of node

File: models/public/rexnet-v1-x1.0/rexnet-v1-x1.0.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+- `B` - batch size
+- `C` - Predicted probabilities for each class in  [0, 1] range
+
+### Converted model
+
+The converted model has the same parameters as the original model.
+
+## Legal Information
+
+The original model is distributed under the following
```
Comment (eaidova) [inline models/public/rexnet-v1-x1.0/rexnet-v1-x1.0.md:69]: original license does not contains license type inside, I suggest directly specify it like in the original repo

File: models/public/rexnet-v1-x1.0/rexnet-v1-x1.0.md (hunk 4 above)
Comment (Anna-Mironova) [reply models/public/rexnet-v1-x1.0/rexnet-v1-x1.0.md]: Updated

File: models/public/rexnet-v1-x1.0/model.yml (hunk 5 above)
Comment (Anna-Mironova) [reply models/public/rexnet-v1-x1.0/model.yml]: Done

File: models/public/rexnet-v1-x1.0/rexnet-v1-x1.0.md (hunk 6 above)
Comment (Anna-Mironova) [reply models/public/rexnet-v1-x1.0/rexnet-v1-x1.0.md]: Done

**Review by eaidova — APPROVED** [review]

File: models/public/rexnet-v1-x1.0/model.yml [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+conversion_to_onnx_args:
+  - --model-path=$dl_dir
+  - --model-name=ReXNetV1
+  - --weights=$dl_dir/rexnetv1_1.0x.pth
+  - --import-module=rexnetv1
+  - --model-param=width_mult=1.0
+  - --input-shape=1,3,224,224
+  - --output-file=$conv_dir/rexnet-v1-x1.0.onnx
+  - --input-names=input.1
+  - --output-names=output
```
Comment (IRDonch) [inline models/public/rexnet-v1-x1.0/model.yml:54]: Maybe you could use `prob`, like with other classification models?

**Review by IRDonch — APPROVED** [review]

**Comment by eaidova** [comment]
Jenkins please retry a build

**Pull request merged**